addons/web/static/src/xml/crash_manager.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can remove the templates coming from base (do it in a move commit before editing this one). Then don't forget to include this one to xml dependencies of those using it previously.